src/Compilers/VisualBasic/Portable/VisualBasicExtensions.vb

```diff
@@ -622,7 +622,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             If symbol IsNot Nothing Then
                 Return StaticCast(Of INamedTypeSymbol).From(symbol.GetModuleMembers())
             Else
-                Return ImmutableArray.Create(Of INamedTypeSymbol)()
+                Return ImmutableArray(Of INamedTypeSymbol).Empty
```

Comment:
📝 The before and after here are exactly the same in practice:

https://github.com/dotnet/corefx/blob/477abf147d8860fae35f975fddafb2711ee42bd4/src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray.cs#L28-L31

Reply:
Not quite, the after uses one less function call in each case. _Itsy perf gain_